CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ project(FatCRM)
 cmake_minimum_required(VERSION 2.8.12) # Feature: PUBLIC keyword in target_link_libraries
 if(COMMAND cmake_policy)
   cmake_policy(SET CMP0003 NEW)
+  cmake_policy(SET CMP0059 OLD)
```

Comment:
oops, I forgot this bit:

if(POLICY CMP0059)
   cmake_policy(SET CMP0059 OLD) # we use ``DEFINITIONS`` as a built-in directory property
endif()